frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,20 @@
-export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
+export enum Status {
```

Comment:
do we want to update the BE variable names to match? `AssessmentStatus -> Status` for `AssessmentType -> UseCase`

in a follow up pr

Reply:
sounds good, will do! i think it might be a good idea to create a ticket to consolidate our types in general sometime soon 🤔